common/src/main/java/com/microsoft/identity/common/internal/result/LocalAuthenticationResult.java

```diff
@@ -200,4 +203,14 @@ public void setRefreshTokenAge(final String refreshTokenAge) {
     public AccessTokenRecord getAccessTokenRecord() {
         return mAccessTokenRecord;
     }
+
+
+    public void setServicedFromCache(Boolean servicedFromCache) {
```

Comment:
nit: add comment, here and every new fields introduced in this PR.

Reply:
Seems like this method is actually not being used - so I'll just remove it

Reply:
Addressed